lighthouse-cli/scripts/run-smoke-tests.sh

```diff
@@ -1,14 +1,15 @@
 #!/usr/bin/env bash
 
-cd lighthouse-cli/test/fixtures && python -m SimpleHTTPServer 9999 &
+cd lighthouse-cli/test/fixtures && python -m SimpleHTTPServer 10200 &
 
 NODE=$([ $(node -v | grep -E "v4") ] && echo "node --harmony" || echo "node")
-config="$PWD/lighthouse-cli/test/fixtures/smoketest-config.json"
-flags="--config-path=$config"
+#config="$PWD/lighthouse-cli/test/fixtures/smoketest-config.json"
+#flags="--config-path=$config"
 
 offline200result="URL responds with a 200 when offline"
 
-$NODE lighthouse-cli $flags http://localhost:9999/online-only.html > results
+# run default lighthouse run against a boring basic page
+$NODE lighthouse-cli --quiet http://localhost:10200/online-only.html > results
```

Comment:
IMO it makes it harder to go through the travis logs and I've not yet debugged a smoketest via its logs.